apps/ewallet_db/lib/ewallet_db/account.ex

```diff
@@ -108,7 +108,8 @@ defmodule EWalletDB.Account do
     end
   end
 
-  @spec avatar_changeset(changeset :: Ecto.Changeset.t(), attrs :: map()) :: Ecto.Changeset.t()
+  @spec avatar_changeset(changeset :: Ecto.Changeset.t() | %Account{}, attrs :: map()) ::
+          Ecto.Changeset.t() | no_return() | none()
```

Comment:
I'm not too familiar with typespec, but isn't `no_return()` and `none()` the same? ([ref](https://hexdocs.pm/elixir/typespecs.html#built-in-types))

Also applies to the remainder of the PR.